Assets/Scripts/Characters/Inventory.cs

```diff
@@ -18,19 +20,25 @@ private void Start()
             items.Add(new ItemBase());
         }
     }
+
     private void Update()
     {
-        if(Input.GetKeyDown(showInventory))
+        if (!player)
         {
-            if(cellContainer.activeSelf)
-            {
-                cellContainer.SetActive(false);
-                Time.timeScale = 1f;
-            }
-            else
+            player = GameObject.FindWithTag("Player");
```

Comment:
You can simplify this by placing this code into `Start()` function

Reply:
If I put this code in the start function, it will be executed once when the player is in the menu and then the inventory will not work.